src/outputs/temp_file_to_be_deleted.py

```diff
@@ -0,0 +1,43 @@
+
+import pandas as pd
+
+from src.utils.wrappers import exception_wrap
+
+
+@exception_wrap
+def combine_dataframes(main_df: pd.DataFrame, mapper_csv_path: pd.DataFrame) -> pd.DataFrame:
+    """
+    Combine two DataFrames using a left join based on specified columns.
+
+    Args:
+        main_df (pandas.DataFrame): The main DataFrame.
+        mapper_csv_path (str): The path to the CSV file for the mapper DataFrame.
+
+    Returns:
+        pandas.DataFrame: The combined DataFrame resulting from the left join.
+    """
+    try:
+        # Read the mapper DataFrame from CSV
+        mapper_df = pd.read_csv(mapper_csv_path)
```

Comment:
For consistency with the rest of the pipeline, we suggest reading this mapper in staging/staging_main.py

Use the read_csv function that we're using for the other mappers (rather than pd.read_csv). This will ensure compatibility with both network and hdfs

Then pass the mapper as a parameter to the outputs module in the same way as we pass read-in dataframes in the other modules.

Reply:
It's done in staging now